drivers/usb/device/usb_dc_native_posix.c

```diff
@@ -351,8 +351,13 @@ int usb_dc_ep_write(const u8_t ep, const u8_t *const data,
 	}
 
 	if (USBIP_EP_ADDR2IDX(ep) == 0) {
-		usbip_send_common(ep, data_len);
-		usbip_send(ep, data, data_len);
+		if (!usbip_send_common(ep, data_len)) {
```

Comment:
It is confusing, I would not expect the function (usbip_send_common) return bool here, what is the benefit to have bool instead of int?

Reply:
We could return number of bytes written or -1 but we need to check that we wrote exact number of bytes. The true indicates that there is no error and exact packet size is sent.

Reply:
Never mind: I had missed the change to the signature.  It'd be nice if error codes associated with the underlying send could be preserved, but this isn't wrong.